styles.css

```diff
@@ -0,0 +1,441 @@
+}
+
+/* Задаем расположение background по проекту */
+.main {
+  position: relative;
+  margin-top: 62px;
+}
+.main__img-background {
+  position: absolute;
+  z-index: 1;
+  width: 100%;
+}
+
+/* Задаем размеры и расположение заголовка */
+.main__title {
+  margin-bottom: 19px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+ .main__h1 {
+  margin: 0 auto;
+  font-size: 90px;
+  line-height: 100px;
+}
+.main__span {
+  font-weight: 500;
+  font-size: 14.7px;
+  line-height: 14px;
+}
+
+/* Задаем расположение изображения в main */
+.main__img-tesla {
+  margin-top: 28px;
+  margin-right: 74px;
+  width: 67%;
+  z-index: 2;
+}
+
+/* Создаем блок для размещения элементов под изображением */
+.main__wrap {
+  display: flex;
+  flex-wrap: wrap;
+  font-family: Montserrat;
+  margin: 0 auto;
+  justify-content: space-between;
+  width: 59%;
+}
+
+/* Блока с параметрами */
+.main__inner {
+  display: flex;
+  justify-content: space-around;
+  width: 50%;
+}
+
+/* Блок  Grid*/
+.main__inner-speed {
+  display: grid;
+}
+.main__inner-road {
+  display: grid;
+  padding-left: 26px;
+}
+
+/* Помещаем картинку в Grid*/
+.main__img-speed {
+  margin-right: 7px;
+  width: 50px;
+  height: 35.5px;
+  grid-column: 1;
+  grid-row: 1/3;
+  align-self: center;
+}
+
+/* Помещаем большой текст в Grid */
+.main__text-big {
+  margin: 0;
+  font-size: 35px;
+  line-height: 35px;
+  grid-column: 2;
+  grid-row: 1;
+  align-self: end;
+}
+
+/* Помещаем средний текст в Grid */
+.main__text-norm {
+  margin: 0;
+  font-size: 20px;
+  grid-column: 3;
+  grid-row: 1;
+  align-self: end;
+  padding-left: 6px;
+}
+
+/* Помещаем маленький текст в Grid */
+.main__text-small {
+  margin: 0;
+  font-weight: 400;
+  font-size: 12px;
+  flex-direction: row;
+  grid-column: 2/4;
+  grid-row: 2;
+}
+
+/* Помещаем картинку в Grid */
+.main__img-road_ipad {
+  display: none;
+}
+.main__img-road_dasktop {
+  margin: 13px 6px 0px 0px;
+  width: 42px;
+  height: 59px;
+  grid-column: 1;
+  grid-row: 1/3;
+}
+
+
+/* Задаем переметры для ссылки тестдрайв */
+.main__div-testdrive {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 43%
+}
+.testdrive__wrapper {
+  display: flex;
+  align-items: center;
+  width: 118px;
+  justify-content: space-between;
+}
+.main__img-testdrive {
+  fill: #000;
+  height: 22px;
+  width: 22px;
+}
+.main__link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  font-size: 13px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #000;
+}
+.main__link:hover {
+  text-shadow: 0px 4px 4px rgba(0, 0, 0,.4);
+}
+
+/* Параметры кнопки */
+.main__button {
+  width: 160px;
+  font-size: 13px;
+  line-height: 36px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #E51937;
+  background: transparent;
+  border: 2px solid #E51937;
+  text-align: center;
+  transition-duration: 0.4s;
+}
+.main__button:hover {
+  background: #E51937;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5575857)